lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/authentication/KerberosAuthentication.java

```diff
@@ -30,6 +30,7 @@
 import java.nio.file.Path;
```

Comment:
Can you please update docs?

Reply:
I'm planning to a have a dedicated PR for docs.